OWNERS.md

```diff
@@ -86,5 +84,7 @@ Thank you, Friends!
 * Lakshmi Kannan ([@lakshmi-kannan](https://github.com/lakshmi-kannan)) - early Stormer. Initial Core platform architecture, scalability, reliability, Team Leadership during the project hard times.
 * Lindsay Hill ([@LindsayHill](https://github.com/LindsayHill)) - ex StackStorm product manager that made a significant impact building an ecosystem we see today.
 * Manas Kelshikar ([@manasdk](https://github.com/manasdk)) - ex Stormer. Developed (well) early core platform features.
+* Nick Maludy ([@nmaludy](https://github.com/nmaludy)) <<nmaludy@gmail.com>>
+  - Community, Core, Systems, Infrastructure, StackStorm Exchange, Puppet deployment. [Case Study](https://stackstorm.com/case-study-encore/).
 * Vineesh Jain ([@VineeshJain](https://github.com/VineeshJain)) - ex Stormer. Community, Tests, Core, QA.
```

Comment:
Hi @nmaludy, the multi-line format with email is suitable if you'd like to move it under the "Maintainers" section.
As we reduced the maintainer responsibilities and if you still would like to maintain the Puppet part only, that would be fine.

Otherwise, in the case of `Friends` or `Contributors` sections, the format needs a small adjustment, see above.